CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/CallConfiguration.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Azure.Communication.Identity;
+using Azure.Communication;
+
+namespace CallAutomation_AppointmentReminder
+{
+    /// <summary>
+    /// Configuration assoociated with the call.
+    /// </summary>
+    public class CallConfiguration
+    {
+        public CallConfiguration()
+        {
+
+        }
+
+        /// <summary>
+        /// The connectionstring of Azure Communication Service resource.
+        /// </summary>
+        public string ConnectionString { get; set; }
+
+        /// <summary>
+        /// The source identity.
+        /// </summary>
+        public string SourceIdentity =>  CreateUser(this.ConnectionString).Result;
```

Comment:
Not sure I would combine this type of behavior with a configuration class. Also, this isn't awaited and it's an async operation.